opts/hosts.go

```diff
@@ -24,6 +24,9 @@ var (
 	DefaultTLSHost = fmt.Sprintf("tcp://%s:%d", DefaultHTTPHost, DefaultTLSHTTPPort)
 	// DefaultNamedPipe defines the default named pipe used by docker on Windows
 	DefaultNamedPipe = `//./pipe/docker_engine`
+	// HostGatewayName defines a special string which users can append to --add-host
+	// to add an extra entry in /etc/hosts that maps host.docker.internal to the host IP
+	HostGatewayName = "host-gateway"
```

Comment:
I think this value should be used directly from [moby](https://github.com/moby/moby/pull/40007/files#diff-65c8668a717635f8e126e71c7213f537R7)